### PR TITLE
benchmarks: use more realistic header counts in benchmark

### PR DESCRIPTION
**Pull request by carl-mastrangelo**

### CODE REVIEW

File: benchmarks/src/jmh/java/io/grpc/netty/HeadersBenchmark.java [hunk 1]
```diff
@@ -53,7 +53,7 @@
  */
 @State(Scope.Benchmark)
 public class HeadersBenchmark {
-  @Param({"10", "20", "50", "100"})
+  @Param({"1", "5", "10", "20"})
```
Comment (nmittler) [inline benchmarks/src/jmh/java/io/grpc/netty/HeadersBenchmark.java]: Err ... 1? That doesn't seem very realistic ... how about 5, 10, 20, 30?

Comment (carl-mastrangelo) [inline benchmarks/src/jmh/java/io/grpc/netty/HeadersBenchmark.java]: 1 is what happens on Trailers (Status-code)

Comment (nmittler) [inline benchmarks/src/jmh/java/io/grpc/netty/HeadersBenchmark.java]: ok, fair enough :)

**Comment by nmittler** [comment]
LGTM

**Pull request merged**